src/tooltip.js

```diff
@@ -75,6 +75,7 @@ class Tooltip extends Component {
     useReactNativeModal: true,
     topAdjustment: 0,
     accessible: true,
+    animationType: 'fade',
```

Comment:
the current default is `'none'` could you update this so it is backward compatible?